api/NavigatorLanguage.json

```diff
@@ -107,16 +102,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/NavigatorLanguage/languages",
           "support": {
             "chrome": {
-              "version_added": "37",
```

Comment:
This bug was real and tracked by https://bugs.chromium.org/p/chromium/issues/detail?id=351715, fixed in Chromium 65: https://storage.googleapis.com/chromium-find-releases-static/a02.html#a029f27a6e98a43e8f1492b85ca18007670d9b28

We could keep a note, but does it belong on `navigator.language` or `navigator.languages`?

I don't know what a web developer would do with this information, so my default suggestion is to remove it.